src/app/settings/page.tsx

```diff
@@ -0,0 +1,146 @@
+                                            <Input placeholder="email@example.com" {...field} />
+                                            </FormControl>
+                                            {errors.email && <FormMessage>{errors.email.message}</FormMessage>}
+                                        </FormItem>
+                                        )}
+                                    />
+                                    <FormField
+                                        control={form.control}
+                                        name="password"
+                                        render={({ field }) => (
+                                        <FormItem>
+                                            <FormLabel>Password</FormLabel>
+                                            <FormControl>
+                                            <Input type="password" placeholder="Password" {...field} />
+                                            </FormControl>
+                                            {errors.password && <FormMessage>{errors.password.message}</FormMessage>}
+                                        </FormItem>
+                                        )}
+                                    />
+
+                                    <div className="flex flex-col">
+                                        <Button type="submit" className="bg-gray-400 text-white px-12 w-12">Save</Button>
+                                    </div>
+                                    
+                                    </form>
+                                </Form>
+                            </div>
+                        ) : (
+                            <div className="flex flex-col text-lg mb-6">
+                                <div>
+                                    <p className="font-bold">Email:</p>
+                                    <p className="text-gray-600">{email}</p>
+                                </div>
+                                <div className="mt-2">
+                                    <p className="font-bold">Password:</p>
+                                    <p className="text-gray-600">{passwordAsterisks}</p>
+                                </div>
+                            </div>
+                        )}
+                        <CardTitle>Preferences</CardTitle>
+                        <div className="items-top flex space-x-2 mt-4">
+                            <Checkbox id="terms1" />
+                            <div className="flex flex-row items-center">
+                                <label
+                                htmlFor="terms1"
+                                className="text-sm leading-none text-gray-600 peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+                                >
+                                I would like to recieve emails from The Giving Heart`&apos;`s optional newsletter.
```

Comment:
I think you can get rid of the back ticks from ``&apos;`` since this is what it looks like on the deployment:
![image](https://github.com/Hack4Impact-UMD/the-giving-heart/assets/68040744/38415cf1-4146-47a9-ab25-3c3f46c617de)